test-executor/src/main/java/com/cognifide/aet/executor/RunnerCacheUpdater.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.executor;
```

Comment:
The `SuiteStatusHandle` and `RunnerCacheUpdater` operate on different caches. Because of that, I thought it would be a good idea to separate those classes. I've also thought about not passing caches directly into the `SuiteRunner`, to have a cleaner code. But if you don't think it would be a problem, then sure, I can remove the `RunnerCacheUpdater`. :)